includes/MirahezeMagicHooks.php

```diff
@@ -350,17 +351,23 @@ public static function onGlobalUserPageWikis( &$list ) {
 
 	/** Removes redis keys for jobrunner */
 	public static function removeRedisKey( string $key ) {
-		global $wmgCacheSettings;
+		global $wgJobTypeConf;
 
-		$redisServer = explode( ':', $wmgCacheSettings['jobrunner']['server'] );
+		if ( !isset( $wgJobTypeConf['default']['redisServer'] ) || !$wgJobTypeConf['default']['redisServer'] ) {
```

Comment:
!$wgJobTypeConf['default']['redisServer'] ?? false for simplification, but not completely necessary here.